content/applications/websites/website/publish/multi_website.rst

```diff
@@ -126,6 +126,14 @@ be redirected to the right website using GeoIP.
 .. seealso::
    - :doc:`/administration/maintain/domain_names`
 
+If you setup an extra website and specify the domain name you will have to
+do three things to get this feature to work:
+1. Setup an A-record in the DNS configuration of your hosting provider so
+that "www.mywebsite2.ext" is redirected to the IP of your Odoo instance.
+2. Create an extra Nginx ``server`` block in which you set the
+``server_name`` as you've entered in the "Website domain" field.
+3. Reload and restart your Nginx instance to use the new ``server`` block.
```

Comment:
RST lists must be surrounded by blank lines and continuation lines of list items must be indented correctly.
See [here](http://runbot86.odoo.com/runbot/static/build/14736289-13-0/logs/build/html/applications/websites/website/publish/multi_website.html#manage-domain-names) how it looks now.
```suggestion

1. Setup an A-record in the DNS configuration of your hosting provider so
   that "www.mywebsite2.ext" is redirected to the IP of your Odoo instance.
2. Create an extra Nginx ``server`` block in which you set the
   ``server_name`` as you've entered in the "Website domain" field.
3. Reload and restart your Nginx instance to use the new ``server`` block.
```